src/Robo/Commands/Setup/DrupalCommand.php

```diff
@@ -26,6 +25,19 @@ class DrupalCommand extends BltTasks {
    */
   public function install() {
 
+    $status = $this->getInspector()->getStatus();
+    $connection = @mysqli_connect(
```

Comment:
This looks like it's reintroducing the regression in #3027. We can't assume that mysql exists on the host running blt commands, much less that it's the same mysql instancing that's hosting Drupal.